src/h5web/vis-packs/core/containers/ComplexVisContainer.tsx

```diff
@@ -37,12 +36,7 @@ function ComplexVisContainer(props: VisContainerProps) {
       <ErrorBoundary
         resetKeys={[dimMapping]}
         FallbackComponent={ErrorFallback}
-        onReset={() => {
-          valuesStore.evict({
-            path: entity.path,
-            selection: getSliceSelection(dimMapping),
-          });
-        }}
+        onError={() => evictCancelledValues()}
```

Comment:
Here is the magic call. It's the same in every container, so it will be trivial to remove the duplication later.